django_html_xml_validator/decorators.py

```diff
@@ -0,0 +1,32 @@
+from functools import wraps
+
+from django.http import HttpResponse
+
+from django_html_xml_validator.middleware import VALIDATION_HEADER
+
+
+def no_html_xml_validation(view_func):
+    """
+    Decorator that prevents a view from being validated.
+    """
+
+    @wraps(view_func)
+    def _wrapped_view_func(request, *args, **kwargs):
+        # Ensure argument looks like a request.
+        if not hasattr(request, "META"):
+            raise TypeError(
+                "skip_html_xml_validation didn't receive an HttpRequest. If you are "
```

Comment:
Hier sollte "a HttpRequest" stehen

Reply:
The code is correct. The "h" in HTTP is pronounced as "aitch", which starts with a vowel, thus "an".